docs/website/docs/dlt-ecosystem/verified-sources/slack.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Slack
+description: dlt verified source for Slack API
+keywords: [slack api, slack verified source, slack]
+---
+
+# Slack
+
+:::info Need help deploying these sources, or figuring out how to run them in your data stack?
+
+[Join our Slack community](https://dlthub-community.slack.com/join/shared_invite/zt-1slox199h-HAE7EQoXmstkP_bTqal65g)
+or [book a call](https://calendar.app.google/kiLhuMsWKpZUpfho6) with our support engineer Adrian.
+:::
+
+[Slack](https://slack.com/) is a popular messaging and collaboration platform for teams and organizations.
+
+This Slack `dlt` verified source and
+[pipeline example](https://github.com/dlt-hub/verified-sources/blob/master/sources/slack_pipeline.py)
+loads data using “Slack API” to the destination of your choice.
+
+Sources and resources that can be loaded using this verified source are:
+
+| Name                  | Description                                                                |
+| --------------------- | -------------------------------------------------------------------------- |
+| slack_source          | Retrives resources conversations, conversations_history and access_logs    |
+| channels_resource     | Retrives all the channels                                                  |
+| get_messages_resource | Retrives all the messages for a given channel                              |
+| access_logs           | Retrives the access logs                                                   |
+
```

Comment:
The users resource is in review and should be merged soon, I think you can add it too.